README.md

```diff
@@ -1042,6 +1043,27 @@ ID of the stp global config. Valid values are default.
 
 Default value: default
 
+#### ios_network_dns
+
+Configure DNS settings for network devices.
+
+#### Properties
+
+The following properties are availible in the `ios_network_dns` type.
+
+##### ip_domain_lookup
+
+Data type: `Optional[Boolean]`
+
+Sets whether the Domain Name Server (DNS) lookup feature should be enabled, by default this should return true.
```

Comment:
As this is an Optional value, this `by default this should return true.` does not make sense.
The example can also be removed as it doesn't demonstrate anything complex.

Reply:
have updated the wording and removed the example